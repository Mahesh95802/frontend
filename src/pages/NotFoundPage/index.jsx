import React from 'react';

const NotFoundPage = () => {
	return(
		<div className="404">
			<h1>404</h1>
			<h2>NotFound</h2>
		</div>
	);
};

export default NotFoundPage;