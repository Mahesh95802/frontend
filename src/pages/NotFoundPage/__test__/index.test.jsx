import React from 'react';
import { render } from '@testing-library/react';
import NotFoundPage from '..';

describe('NotFoundPage', () => {
    it('should render the page', () => {
        const { asFragment } = render(<NotFoundPage />);
        expect(asFragment()).toMatchSnapshot();
    });
});