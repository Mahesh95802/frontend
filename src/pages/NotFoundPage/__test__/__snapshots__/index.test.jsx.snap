// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should render the page 1`] = `
<DocumentFragment>
  <div
    class="404"
  >
    <h1>
      404
    </h1>
    <h2>
      NotFound
    </h2>
  </div>
</DocumentFragment>
`;
